src/NzbDrone.Core/Indexers/Definitions/AnimeBytes.cs

```diff
@@ -497,7 +497,7 @@ public AnimeBytesSettings()
         [FieldDefinition(1, Label = "Passkey", Privacy = PrivacyLevel.Password, Type = FieldType.Password, HelpText = "Site Passkey")]
         public string Passkey { get; set; }
 
-        [FieldDefinition(1, Label = "Username", HelpText = "Site Username")]
+        [FieldDefinition(1, Label = "Username", Advanced = false, HelpText = "Site Username", Type = FieldType.Textbox, Privacy = PrivacyLevel.UserName)]
```

Comment:
No need to define `Advanced = false` as this is the default for fields (same for all of them)